doi-requests-commons/src/main/java/no/unit/nva/doi/requests/userdetails/UserDetails.java

```diff
@@ -0,0 +1,27 @@
+package no.unit.nva.doi.requests.userdetails;
+
+import static nva.commons.utils.RequestUtils.getRequestContextParameter;
+
+import com.fasterxml.jackson.core.JsonPointer;
+import nva.commons.handlers.RequestInfo;
+
+public final class UserDetails {
+
+    public static final String ROLE = "role";
```

Comment:
Functionality taken out from `FindDoiRequestHandler` to re-use in `CreateRequestHandler`.